app/screens/settings/advanced_settings/advanced_settings.js

```diff
@@ -43,56 +43,37 @@ class AdvancedSettings extends PureComponent {
     }
 
     clearOfflineCache = preventDoubleTap(() => {
-        const {actions, intl} = this.props;
+        const {formatMessage} = this.props.intl;
 
         Alert.alert(
-            intl.formatMessage({id: 'mobile.advanced_settings.reset_title', defaultMessage: 'Reset Cache'}),
-            intl.formatMessage({id: 'mobile.advanced_settings.reset_message', defaultMessage: '\nThis will reset all offline data and restart the app. You will be automatically logged back in once the app restarts.\n'}),
+            formatMessage({id: 'mobile.advanced_settings.delete_title', defaultMessage: 'Delete Documents & Data'}),
+            formatMessage({id: 'mobile.advanced_settings.delete_message', defaultMessage: '\nThis will reset all offline data and restart the app. You will be automatically logged back in once the app restarts.\n'}),
             [{
-                text: intl.formatMessage({id: 'mobile.advanced_settings.reset_button', defaultMessage: 'Reset'}),
-                onPress: () => actions.purgeOfflineStore(),
-            }, {
-                text: intl.formatMessage({id: 'channel_modal.cancel', defaultMessage: 'Cancel'}),
+                text: formatMessage({id: 'channel_modal.cancel', defaultMessage: 'Cancel'}),
+                style: 'cancel',
                 onPress: () => true,
-            }]
+            }, {
+                text: formatMessage({id: 'mobile.advanced_settings.delete', defaultMessage: 'Delete'}),
+                style: 'destructive',
+                onPress: this.purgeDocumentsAndData,
+            }],
+            {cancelable: false},
```

Comment:
Is this here on purpose? You're still declaring cancel text above, but you've changed it to not be cancelable

Reply:
Cancelable false means that it cannot be dismissed on Android by tapping outside the alert box

Reply:
Ah, okay